.editorconfig

```diff
@@ -0,0 +1,29 @@
+# SPDX-FileCopyrightText: 2025 SAP SE or an SAP affiliate company
+# SPDX-License-Identifier: Apache-2.0
+
+# SPDX-FileCopyrightText: SAP SE or an SAP affiliate company
+# SPDX-License-Identifier: Apache-2.0
```

Comment:
That shouldn't happen. Probably changing the content of reuse confused it. You only need and want one.